packages/react-native/React/Views/RCTTVView.m

```diff
@@ -458,23 +458,43 @@ - (RCTTVView *)getViewById:(NSNumber *)viewId {
 }
 
 - (void)setNextFocusUp:(NSNumber *)nextFocusUp {
-  self->_nextFocusUp = [self getViewById: nextFocusUp];
-  [self enableDirectionalFocusGuides];
+  if (self.focusGuideUp != nil && nextFocusUp == nil) {
```

Comment:
This change makes sense! Could you please have a look at the Fabric implementation as well, to see if similar adjustments are needed?

https://github.com/react-native-tvos/react-native-tvos/blob/13afea00a2495858266ec83416b86930d464033f/packages/react-native/React/Fabric/Mounting/ComponentViews/View/RCTViewComponentView.mm#L1074

Reply:
Done

Reply:
Just a note, struggled to get nextFocus working on rn-tester fabric before and after the change - can you check it out also?

Reply:
Yes, there is a nextFocus example in RNTester that works fine with Fabric. I'll test this and verify on tvOS.